.github/workflows/build_linux_packages.yml

```diff
@@ -125,6 +128,13 @@ jobs:
           python build/indexer.py -f '*.log' build/logs/
           sed -i 's,a href="..",a href="../../index-${{env.AMDGPU_FAMILIES}}.html",g' build/logs/index.html
 
+      - name: Configure AWS Credentials
+        if: ${{ !cancelled() }}
+        uses: aws-actions/configure-aws-credentials@ececac1a45f3b08a01d2dd070d28d111c5fe6722 # v4.1.0
+        with:
+          aws-region: us-east-2
+          role-to-assume: arn:aws:iam::692859939525:role/therock-artifacts
```

Comment:
Credentials where configured earlier as part of this job, thus this shouldn't be needed here.

Reply:
The credentials expires after some time, If I do not do second one, I got something like that:` The provided token has expired.`

Reply:
Do you know the default lifetime? I guess we could rather try to move configuring the credentials down right before there step where the token is used for the first time. Right now the token is obtained even before sources are fetched (which takes some minutes).

Reply:
AFAIK, it is about one hour -  I put it right before the Build Project where `teatime.py` is in action in the building. I am doing the test now.

Reply:
I tested this flag as below:
`role-duration-seconds: 14400
`
It requires further privileges otherwise I got this 
```
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Assuming role with OIDC
Error: Could not assume role with OIDC: The requested DurationSeconds exceeds the MaxSessionDuration set for this role.
```

Reply:
By the way, we need this flag anyway:
`role-duration-seconds: 14400 `
Since the building time approximately two hours and sometimes more, token are expired during the Building Step. But again there is a privilege issue I come across. 

If we overcome the privilege issue, we do not need to authenticate two time as well.

Reply:
Sounds like we need to edit that MaxSessionDuration on the cloud project side.

Reply:
I agree

Reply:
Do you know anyone I need to get in touch for that?